<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>КВК Групп</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44,80,190);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QHBoxLayout" name="headerLayout">
      <item>
       <widget class="QLabel" name="logoLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="pixmap">
         <pixmap>../../../.designer/media/cvc.PNG</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>24</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>CVC-group</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QPushButton" name="backButton">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #E74C3C; color: white; border: #E74C3C;</string>
      </property>
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageLogin">
       <layout class="QVBoxLayout" name="loginLayout">
        <item>
         <widget class="QLabel" name="loginTitle">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Авторизация</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayoutLogin">
          <item row="0" column="0">
           <widget class="QLabel" name="labelLogin">
            <property name="text">
             <string>Логин:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="loginInput"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelPassword">
            <property name="text">
             <string>Пароль:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="passwordInput">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnLogin">
          <property name="styleSheet">
           <string notr="true">background-color: #67BA80;
border: #67BA80;</string>
          </property>
          <property name="text">
           <string>Войти</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageMain">
       <layout class="QVBoxLayout" name="mainPageLayout">
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="styleSheet">
           <string notr="true">QListWidget {
    background-color: white;
    outline: 0;
}

QListWidget::item {
    border: 1px solid #ddd;
    padding: 14px;
    margin: 5px;
    min-height: auto;
    height: auto;
}

QListWidget::item:selected {
    background-color: #f0f0f0;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonLayout">
          <item>
           <widget class="QPushButton" name="btnAdd">
            <property name="styleSheet">
             <string notr="true">background-color: #67BA80;
border: #67BA80;</string>
            </property>
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnLogout">
            <property name="styleSheet">
             <string notr="true">background-color: #E74C3C; color: white; border: #E74C3C;</string>
            </property>
            <property name="text">
             <string>Выйти</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="employeePage">
       <layout class="QGridLayout" name="employeeLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="positionLabel">
          <property name="text">
           <string>Должность:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="employeeNameLabel">
          <property name="text">
           <string>ФИО:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="employeeNameEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="positionCombo">
          <item>
           <property name="text">
            <string>Менеджер</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Разработчик</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Аналитик</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="departmentCombo">
          <item>
           <property name="text">
            <string>IT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Маркетинг</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Продажи</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="contactsLabel">
          <property name="text">
           <string>Контакты:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="contactsEdit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="departmentLabel">
          <property name="text">
           <string>Отдел:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="saveEmployeeButton">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="deleteEmployeeButton">
          <property name="text">
           <string>удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="storagePage">
       <layout class="QGridLayout" name="storageLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="productNameLabel">
          <property name="text">
           <string>Название:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="productNameEdit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="articulLabel">
          <property name="text">
           <string>Артикул:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="articulEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="categoryLabel">
          <property name="text">
           <string>Категория:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="locationLabel">
          <property name="text">
           <string>Место хранения:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="locationEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="categoryCombo">
          <item>
           <property name="text">
            <string>Электроника</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Офис</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Хозтовары</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="quantitySpin"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="quantityLabel">
          <property name="text">
           <string>Количество:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="saveStorageButton">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="deleteStorageButton">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="partnerPage">
       <layout class="QGridLayout" name="partnerLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="companyNameLabel">
          <property name="text">
           <string>Название компании:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="directorEdit"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="companyNameEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="partnerPhoneEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="partnerPhoneLabel">
          <property name="text">
           <string>Телефон:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="directorLabel">
          <property name="text">
           <string>Контактное лицо:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="savePartnerButton">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="deletePartnerButton">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
